cmd/cosign/cli/sign/sign.go

```diff
@@ -246,14 +256,21 @@ func signDigest(ctx context.Context, digest name.Digest, payload []byte, ko opti
 		s = irekor.NewSigner(s, rClient)
 	}
 
-	ociSig, _, err := s.Sign(ctx, bytes.NewReader(payload))
-	if err != nil {
-		return err
-	}
+	ociSigs := make([]oci.Signature, len(payloads))
+	b64sigs := make([]string, len(payloads))
 
-	b64sig, err := ociSig.Base64Signature()
-	if err != nil {
-		return err
+	for i, payload := range payloads {
```

Comment:
can we use a different variable name than `payload`? we're shadowing the input arg.